app/server.py

```diff
@@ -7,6 +9,47 @@ def refresh():
     writer.update_calendars()
     return '', 200
 
+@app.route('/login', methods=['GET', 'POST'])
+def login():
+    form = forms.LoginForm()
+    flask.flash(form.validate_on_submit())
```

Comment:
Presumably.  I was trying to debug and forgot this but I never actually saw the flash messages anywhere